tests/factory_tests.py

```diff
@@ -140,6 +140,34 @@ def test_one_arg_iso_str(self):
 
         assertDtEqual(self.factory.get(dt.isoformat()), dt.replace(tzinfo=tz.tzutc()))
 
+    def test_one_arg_iso_calendar(self):
```

Comment:
Solid tests, but we should also add the two mentioned in the Python docs: https://docs.python.org/3/library/datetime.html#datetime.date.isocalendar